spanner/spanner_samples.rb

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def create_instance project_id:, instance_id:
+def create_instance project_id, instance_id, instance_config_id = "regional-us-central1"
   # [START spanner_create_instance]
   # project_id  = "Your Google Cloud project ID"
   # instance_id = "Your Spanner instance ID"
```

Comment:
Since `instance_config_id` must now be provided to the code and is no longer hardcoded below, please document it similar to how we are documenting `project_id` and `instance_id`.